doc/architecture/sensor_view_configuration.adoc

```diff
@@ -0,0 +1,15 @@
+= Sensor view configuration
+
+The sensor view configuration configures the sensor view.
+There are two sources for sensor-view-configuration data.
+
+Sensor-view-configuration data can be provided by the environment simulation.
+In this case, the data is intended for the automatic configuration of an environment simulation in order to supply the necessary input to a sensor model.
+Sensor-view-configuration data is not intended to provide parameters to a generic sensor model. 
+
+Sensor-view-configuration data can also be provided by the sensor model to the environment simulation.
```

Comment:
```suggestion
Sensor-view-configuration data can also be provided by the sensor model to the simulation environment.
```
same as above or "environment simulator"

Reply:
No change, because we agreed in CCB Meeting, 04.08.2021 that "environment simulation" is correct.